emission/analysis/classification/inference/mode/rule_engine.py

```diff
@@ -187,8 +187,10 @@ def collapse_modes(section_entry, modes):
     start and end points. This method merges the list of entries returned by
     the GIS into one, potentially using speed information 
     """
-    train_mode_list = ['funicular', 'miniature', 'rail', 'railway',
-        'light_rail', 'subway', 'monorail', 'tram', 'aerialway', ]
+    # train_mode_list = ['funicular', 'miniature', 'rail', 'railway',
+    #     'light_rail', 'subway', 'monorail', 'tram', 'aerialway', ]
+    train_mode_list = ['funicular', 'miniature', 'rail', 'railway', 'light_rail', 'monorail', 
```

Comment:
just to confirm my own understanding here: with this change, you are removing `subway` and `tram` from the list of modes that we should collapse to `train`. What about `track`, the mode that originally started this entire discussion? https://github.com/e-mission/e-mission-docs/issues/426

Reply:
Well, I didn't encounter this issue but I should definitely put "tracks" back on this list, just in case.